manifests/p/Puppet/puppet-bolt/3.12.0/Puppet.puppet-bolt.locale.en-US.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Puppet.puppet-bolt
+PackageVersion: 3.12.0
+PackageLocale: en-US
+Publisher: Puppet, Inc
```

Comment:
```suggestion
Publisher: Puppet, Inc.
```

Reply:
Oh, how did I miss that haha, thank you.